pmg/user_management.py

```diff
@@ -64,6 +64,33 @@ def user_committee_alert(committee_id):
 
     return redirect(request.values.get('next', '/'))
 
+@app.route('/user/follow/committee/<int:committee_id>', methods=['POST'])
+def user_follow_committee(committee_id):
+    if current_user.is_authenticated() and request.method == 'POST':
+        u = current_user.follow_committee(Committee.query.get(committee_id))
+
+        if u is not None:
+            db.session.add(u)
+            db.session.commit()
+            ga_event('user','follow-committee','cte-follow-committee')
+        else:
+            flash('Committee %s cannot be followed.' % committee_id)
+
+    return redirect(request.values.get('next', '/'))
```

Comment:
I suggest this instead, which ensures that the user goes back to their request paging if AJAX fails for some reason. We're not using the `next` param in any case.

```
return redirect(request.headers.get('referer', '/'))
```